src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -0,0 +1,887 @@
+    initialValues,
+    savedOrderData,
+    targetStep: currentStep,
+  } = useLoadOrder(props)
+
+  const isOffer = order.mode === "OFFER"
+
+  const {
+    isArtsyShipping,
+    requiresArtsyShippingTo,
+    shippingQuotesEdges,
+    selectedShippingQuoteId: savedSelectedShippingQuoteId,
+  } = savedOrderData
+
+  const advanceToPayment = useCallback(() => {
+    props.router.push(`/orders/${props.order.internalID}/payment`)
+  }, [props.router, props.order.internalID])
+
+  // Reset fulfillment details on load if artsy shipping to refresh shipping quotes
+  // Note: exact current behavior is documented in notion. On load of shipping page, it
+  // re-saves the order using the default address (not what is selected) which triggers a
+  // shipping quote refresh. the *selected shipping quote is not reset.*
+  useEffect(() => {
+    if (
+      savedOrderData.fulfillmentType === FulfillmentType.SHIP &&
+      isArtsyShipping
+    ) {
+      handleSubmitFulfillmentDetails(initialValues.fulfillmentDetails)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const previousActiveStep = usePrevious(currentStep)
+
+  // Advance to payment when the order is ready to proceed unless it loaded that way
+  useEffect(() => {
+    if (
+      currentStep === "ready_to_proceed" &&
+      previousActiveStep !== "ready_to_proceed"
+    ) {
+      advanceToPayment()
+    }
+  }, [advanceToPayment, currentStep, previousActiveStep])
+
+  const [selectedShippingQuoteId, setSelectedShippingQuoteId] = useState<
+    string | undefined
+  >(savedSelectedShippingQuoteId)
+
+  // const selectShipping = async (editedAddress?: MutationAddressResponse) => {
+  // ...
+
+  //   TODO: Tracking
+  //     trackEvent({
+  //       action: ActionType.errorMessageViewed,
+  //       context_owner_type: OwnerType.ordersShipping,
+  //       context_owner_id: props.order.internalID,
+  //       title: "An error occurred",
+  //       message:
+  //         "Something went wrong. Please try again or contact orders@artsy.net.",
+  //       error_code: null,
+  //       flow: "user selects a shipping option",
+  //     })
+
+  //     props.dialog.showErrorDialog()
+  //   }
+  // }
+
+  const handleSubmitError = useCallback(
+    (error: { code: string; data: string | null }) => {
+      logger.error(error)
+      const parsedData = error.data ? JSON.parse(error.data) : {}
+      if (
+        error.code === "missing_region" ||
+        error.code === "missing_country" ||
+        error.code === "missing_postal_code"
+      ) {
+        trackEvent({
+          action: ActionType.errorMessageViewed,
+          context_owner_type: OwnerType.ordersShipping,
+          context_owner_id: props.order.internalID,
+          title: "Invalid address",
+          message:
+            "There was an error processing your address. Please review and try again.",
+          error_code: error.code,
+          flow: "user submits a shipping option",
+        })
```

Comment:
Can we move all of the tracking function calls used throughout the shipping file into their own `useShippingTracking` file? Something like this: https://github.com/artsy/force/blob/main/src/Apps/Auction/Hooks/useAuctionTracking.ts

Reply:
Doing it in this PR (which would be very quick to implement) could be a quick way to clean up a lot of secondary noise that's sprinkled throughout. 

Def not blocking, just a suggestion to avoid follow-ups for trivial (yet valuable) readability stuff in such a complex PR that could likely be returned to as a reference.